util/http.py

```diff
@@ -0,0 +1,27 @@
+"""Utilities for dealing with HTTP."""
+from __future__ import annotations
+
+from gzip import compress
+
+__all__ = ['gzip']
+
+
+def gzip(data: str | bytes, max_size: int = 512, encoding='utf-8') -> tuple[bytes, bool]:
+    """
```

Comment:
Technically this is code that can be used for more than "we're about to sling out a http body".

`util/text.py` ?

Reply:
moved